migration/src/lib/drupal-content.js

```diff
@@ -137,10 +147,36 @@ class DrupalContent {
         });
         return convertedItem;
     }
-
```

Comment:
Can we just get a comment here that indicates that the following code is about to start loading in the actual content using the methods above?